cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -61,5 +58,15 @@
                 <p class='field-message' id='time-limit-description'><%- gettext('Learners see warnings when 20% and 5% of the allotted time remains. You can grant learners extra time to complete the exam through the Instructor Dashboard.') %></p>
             </ul>
         </div>
+        <div class='exam-review-rules-list-fields is-hidden' id='id_exam_review_rules_div'>
```

Comment:
Nit: don't use underscores in id/class names. My preference would be to not include ids at all. Also, don't name ids as "foo_div" because that isn't semantic. In this case, I think the class is sufficient.